### ISSUE TITLE
failed on go1.13

### ISSUE THREAD

**Issue by FelixSeptem** [golang-migrate/migrate#302]
**Describe the Bug**
```
>>> go get -u -d github.com/golang-migrate/migrate/cmd/migrate
go: finding github.com/golang-migrate/migrate/cmd/migrate latest
go: finding github.com/golang-migrate/migrate/cmd latest
go get github.com/golang-migrate/migrate/cmd/migrate: module github.com/golang-migrate/migrate@upgrade (v3.5.4+incompatible) found, but does not contain package github.com/golang-migrate/migrate/cmd/migrate
```

**Steps to Reproduce**
my environment:
```shell
GO111MODULE="on"
GOARCH="amd64"
GOBIN=""
GOCACHE="/Users/myname/Library/Caches/go-build"
GOENV="/Users/myname/Library/Application Support/go/env"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="darwin"
GONOPROXY="*.mycompany.com"
GONOSUMDB="*.mycompany.com"
GOOS="darwin"
GOPATH="/Users/haowen/.gvm/pkgsets/go1.13/global"
GOPRIVATE="*.mycompany.com"
GOPROXY="https://goproxy.cn,direct"
GOROOT="/Users/mycompany/.gvm/gos/go1.13"
GOSUMDB="sum.golang.org"
GOTMPDIR=""
GOTOOLDIR="/Users/mycompany/.gvm/gos/go1.13/pkg/tool/darwin_amd64"
GCCGO="gccgo"
AR="ar"
CC="clang"
CXX="clang++"
CGO_ENABLED="1"
GOMOD="/dev/null"
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -fdebug-prefix-map=/var/folders/h1/9ltt1sp94_x8yzjxjjw5ppbr0000gp/T/go-build058717330=/tmp/go-build -gno-record-gcc-switches -fno-common"
```

**Expected Behavior**
get repo successful

**Comment by dhui** [comment]
Seems related to https://github.com/golang-migrate/migrate/pull/257

**Comment by FelixSeptem** [comment]
`go get -tags 'mysql' -u github.com/golang-migrate/migrate/v4/cmd/migrate/`
this command raise a new problem with:
```shell
go get: github.com/kshvakov/clickhouse@v1.3.5 updating to
	github.com/kshvakov/clickhouse@v1.3.12: parsing go.mod:
	module declares its path as: github.com/ClickHouse/clickhouse-go
	        but was required as: github.com/kshvakov/clickhouse
```
looks like caused by module define with repo `github.com/kshvakov/clickhouse`

**Comment by Jerome1337** [comment]
Same problem this morning

**Comment by bxcodec** [comment]
I've submitted a PR at #305 hopefully can be approved and merged ASAP.
This is happening because @kshvakov move his package to a new organization.

Anyway, is there any short-term solution? Since waiting for PR to be approved may takes time 🤔

**Comment by dhui** [comment]
Also related to: https://github.com/golang-migrate/migrate/issues/303

**Comment by FelixSeptem** [comment]
```shell
>>> go get -tags 'mysql' -u github.com/golang-migrate/migrate/v4/cmd/migrate/
go get: github.com/kshvakov/clickhouse@v1.3.5 updating to
	github.com/kshvakov/clickhouse@v1.3.12: parsing go.mod:
	module declares its path as: github.com/ClickHouse/clickhouse-go
	        but was required as: github.com/kshvakov/clickhouse
```
different error but still broken

**Comment by bxcodec** [comment]
@FelixSeptem  try to fetch the master version

```bash
go get -tags 'postgres' -u github.com/golang-migrate/migrate/v4/cmd/migrate@master
```

**Comment by bxcodec** [comment]
It happened since the go get will fetch the last released version. That still using the old dependencies

**Comment by bxcodec** [comment]
Can you release a new version @dhui?
And also some announcements to tell people to update the version. I'm afraid this will affected the old version.

**Comment by FelixSeptem** [comment]
```
go get -tags 'mysql' -u github.com/golang-migrate/migrate/v4/cmd/migrate@master
```
this command really help for me, https://github.com/golang-migrate/migrate/issues/302#issuecomment-557411691 this is the true reason for above error, may solved after a new release tag made.

**Comment by dhui** [comment]
Released [v4.7.1](https://github.com/golang-migrate/migrate/releases/tag/v4.7.1)
